packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -2282,17 +2286,30 @@ describe(componentName, () => {
     expect(tableElement).not.toHaveClass(`${blockClass}__table-grid-active`);
   });
 
+  const sharedFilterGridProps = {
+    gridTitle: 'Data table title',
+    gridDescription: 'Additional information if needed',
+    useDenseHeader: false,
+    emptyStateTitle: 'No filters match',
+    emptyStateDescription:
+      'Data was not found with the current filters applied. Change filters or clear filters to see other results.',
+  };
+
   it('should test basic interactions of filter panel', async () => {
+    const updatedFilterProps = { ...filterProps };
+    // Removing certain properties to test default function parameters in FilterPanel
+    delete updatedFilterProps.panelTitle;
+    delete updatedFilterProps.closeIconDescription;
+    delete updatedFilterProps.updateMethod;
+    delete updatedFilterProps.primaryActionLabel;
+    delete updatedFilterProps.secondaryActionLabel;
+    delete updatedFilterProps.onPanelOpen;
+    delete updatedFilterProps.onPanelClose;
```

Comment:
would it be easier to just update `updatedFilterProps` to be the props would want and use `...rest` to remove the un-needed props? or since it looks like it's being used a few times just declare a different variable without those particular props? just seeing if it's possible to cut down on redundancy

Reply:
Agreed, I'll update this ^